api_tests/ledger_query_service_conf.js

```diff
@@ -0,0 +1,38 @@
+class LedgerQueryServiceConf {
+    constructor(host, port) {
+        this.host = host;
+        this.port = port;
+    }
+
+    url() {
+        return "http://" + this.host + ":" + this.port;
+    }
+
+    poll_until_matches(chai, query_url) {
+        return new Promise((final_res, rej) => {
+            chai.request(query_url)
+                .get("")
+                .end((err, res) => {
+                    if (err) {
+                        return rej(err);
+                    }
+                    res.should.have.status(200);
+                    if (res.body.matches.length !== 0) {
+                        final_res(res.body);
+                    } else {
+                        setTimeout(() => {
+                            this.poll_until_matches(chai, query_url).then(
+                                result => {
+                                    final_res(result);
+                                }
+                            );
+                        }, 200);
+                    }
+                });
+        });
+    }
+}
+
+module.exports.ledger_query_service_conf = (host, port) => {
```

Comment:
❌ Let's keep it homogenous, please name it `create`, same than `comit_node_conf`.

Reply:
Yes, I actually wanted to get your feedback on this. Will do, I think it's cleaner and more readable than `ledger_query_service_conf.ledger_query_service_conf` :-)